### ISSUE TITLE
MiDaS-v2 was trained using the median + MAE

### ISSUE THREAD

**Issue by zw1992** [intel-isl/MiDaS#33]
Thanks for your work!  In the paper proposeMiDaS-v2 was trained using the median + MAD,What's special about the implementation details?I implementationit but the training result is terrible than using the former loss.( I remove the relu at the end 
 of the net )Maybe I make some mistakes.Could you release your implementation of the loss function?

**Comment by LeeJZh** [comment]
The same problem. Could the arthur share your new loss function? Maybe along with training code?

**Comment by ranftlr** [comment]
Please see here for code: https://gist.github.com/ranftlr/1d6194db2e1dffa0a50c9b0a9549cbd2

We've never tried this loss without the ReLU at the end, it is possible that this influences the training dynamic.

**Comment by ranftlr** [comment]
Closing due to inactivity.

**Comment by mzy97** [comment]
> Please see here for code: https://gist.github.com/ranftlr/1d6194db2e1dffa0a50c9b0a9549cbd2
> 
> We've never tried this loss without the ReLU at the end, it is possible that this influences the training dynamic.

Hi, in paper, you said the median value comes from all valid pixels, but in the loss function below, medians were driven from all pixels. In case of the GT depth map is relatively sparse, will this code cause some issue?
https://gist.github.com/ranftlr/1d6194db2e1dffa0a50c9b0a9549cbd2#file-loss-py-L32

**Comment by 519174419** [comment]
> The same problem
